packages/react-instantsearch-core/src/index.js

```diff
@@ -48,3 +48,4 @@ export { default as connectStats } from './connectors/connectStats';
 export {
   default as connectToggleRefinement,
 } from './connectors/connectToggleRefinement';
+export { default as connectQueryRules } from './connectors/connectQueryRules';
```

Comment:
I think the export are sorted for the connectors, easier to navigate inside them.